src/components/ToDo.js

```diff
@@ -0,0 +1,33 @@
+import "./ToDo.css";
+
+function ToDo({ name, onDelete, onToggle, isDone }) {
+  function handleDeleteToDo() {
+    onDelete(name);
+  }
+
+  function togglePendingButton() {
+    onToggle(name);
+  }
+
+  return (
+    <div className="list-container">
+      <ul clasName="list">
+        <li className="list-items">
+          <button className="RemoveButton" onClick={handleDeleteToDo}>
+            {" "}
```

Comment:
You  don't need this `{" "}`, nor the one after the X. It seems sometimes this happens with CodeSandbox.